pkg/apis/application/v1alpha1/chart_types.go

```diff
@@ -74,8 +78,18 @@ type ChartSpec struct {
 	Version string `json:"version"`
 }
 
+// +k8s:openapi-gen=true
+type ChartSpecInstall struct {
+	// +kubebuilder:validation:Optional
+	// +nullable
+	// SkipCRDs decides whether to install CRDs that might be supplied with the chart
+	SkipCRDs bool `json:"skipCrds,omitempty"`
```

Comment:
There is a bit of a chance of a misinterpretation here. "skip" to me indicates that a CRD _will not_ be installed. However the description seems to indicate the opposite.

This could be resolved by making the description more explicit. Or by renaming the field.

Also it should be mentioned what the default is (if not set). This may be obvious in a Go context, but not in a CRD context.

```suggestion
	// SkipCRDs when true decides that CRDs which are supplied with the chart are not installed. Default: false.
	SkipCRDs bool `json:"skipCrds,omitempty"`
```

or

```suggestion
	// InstallCRDs decides that CRDs which are supplied with the chart will be installed (default). Set to false to omit CRD installation.
	InstallCRDs bool `json:"installCrds,omitempty"`
```